docs/source/guide/pt-4-low-level.md

```diff
@@ -0,0 +1,128 @@
+
+- Twirling CZ and CNOT gates in the circuit;
+- Executing the modified circuit.
+- Estimate expectation value by averaging over randomized twirling circuits
+
+## Twirling the Input Circuit
+To twirl about particular gates, we need the Pauli group for those gates. These groups are stored as lookup tables, in {attr}`mitiq.pt.pt.CNOT_twirling_gates` and {attr}`mitiq.pt.pt.CZ_twirling_gates`, so that we can randomly select a tuple from the group. Now we're ready to twirl our gates.
+
+First let's define our circuit:
+```{code-cell} ipython3
+from cirq import LineQubit, Circuit, CZ, CNOT
+
+a, b, c, d  = LineQubit.range(4)
+circuit = Circuit(
+    CNOT.on(a, b),
+    CZ.on(b, c),
+    CNOT.on(c, d),
+)
+
+print(circuit)
+```
+Now, we can see what happens when we apply the PT functions, through {func}`.twirl_CNOT_gates()` and the subsequent {func}`.twirl_CZ_gates()`
+```{code-cell} ipython3
+from mitiq import pt
+
+circuit_to_twirl = circuit.copy()
+CNOT_twirled_circuits = pt.twirl_CNOT_gates(circuit_to_twirl, num_circuits=10)
+twirled_circuits = [
+    pt.twirl_CZ_gates(c, num_circuits=1)[0] for c in CNOT_twirled_circuits
+]
+print(CNOT_twirled_circuits[0:1])
+print(twirled_circuits[0:1])
+```
+~~We see that we return lists of the randomly twirled circuits, and so we must take a simple average over their expectation values.~~
+
+
+
+## Verifying Pauli Twirling Works
+
+It is worth checking if the technique does tailor the noise as expected. For this
+demonstration, we compare the off-diagonal terms in the Pauli Transfer Matrix (PTM)
+{cite}`Hashim_2021_PRX` of the original input circuit, the noisy input circuit as well
+as the averaged PTM of the different possible Pauli twirled circuits. A detailed discussion
+on the PTM is available in [theory section of PT](pt-5-theory.md).
+
+Now, instead of using the ideal input circuit, we use PT on the noisy circuit with the
+default option for the number of twirled circuits. 
+
+```{code-cell} ipython3
+from mitiq import pt
+tailored_circuit_noisy = pt.pauli_twirl_circuit(
+    circuit = noisy_circuit
+)
+```
+The PTM of the ideal circuit is in the following code block. 
+
+The PTM of the noisy circuit is in the following code block. 
+
+The PTM of the output from PTM is in the following code block. 
+
+
+~~Pauli Twirling doesn't require running the circuit at different noise levels or with different noise models. It applies a randomized sequence of Pauli operations within the same quantum circuit and averages the results to reduce the effect of the noise.~~
+
+```{code-cell} ipython3
+import numpy as np
+from typing import cast
+
+average = cast(float, np.average(expvals))
+print(average)
+```
+
+Keep in mind, ths code is for illustration and that the noise level, type of noise (here amplitude damping), and the observable need to be adapted to the specific experiment.
+
+
+On a real backend, they have a different sensitivity to noise. The core idea of the PT technique is that,
```

Comment:
If one were to demonstrate an example of PT + some other mitigation technique, is it preferable to 

1. introduce a separate tutorial?
2. Or demonstrate it in this section to keep it consistent with what @Aaron-Robertson originally intended?